native/cocos/audio/android/PcmAudioService.cpp

```diff
@@ -90,7 +95,10 @@ bool PcmAudioService::init(AudioMixerController *controller, int numChannels, in
     if (numChannels > 1) {
         channelMask = SL_SPEAKER_FRONT_LEFT | SL_SPEAKER_FRONT_RIGHT;
     }
-
+#if CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+    // TODO(hack): 1 channel must be set, there is a problem with dual channels
+    numChannels = 1;
```

Comment:
If reset numChannels to 1, the `channelMask` should be changed too. At line 95,96

Reply:
This is just a hack, just because of the openharmony system itself, there is not much difference between playing the left channel and the right channel. I feel better if I don't change it now